queue_job/models/queue_job.py

```diff
@@ -418,6 +418,61 @@ def requeue_stuck_jobs(self, enqueued_delta=5, started_delta=0):
         ).requeue()
         return True
 
+    def fail_dead_jobs(self, started_delta, force_low_delta=False):
+        """Set as failed job started since too long ago.
+
+        Workers can be dead without anyone noticing
+        Dead workers stuck the channel and provoke
+        famine.
+
+        This function, mark jobs started longtime ago
+        as failed.
+
+        Cause of death can be CPU Time limit reached
+        a SIGTERM, a power shortage, we can't know, etc.
+
+        This mechanism should be very exceptionnal.
+        It may help, for instance, if someone forget to configure
+        properly his system.
```

Comment:
```suggestion
        This function marks jobs started longtime ago
        as failed.

        Cause of death can be CPU Time limit reached
        a SIGTERM, a power shortage, etc. We can't know.

        This mechanism should be very exceptional.
        It may help, for instance, if someone forgot to configure
        properly his system.
```